index.js

```diff
@@ -139,6 +144,17 @@ module.exports = function (zipPath, opts, cb) {
                 return done(err)
               })
             }
+            // Attempt to delete an existing symlink before attempting to write
+            function overwriteExistingSymlink () {
+              fs.unlink(dest, function (err) {
+                if (err) {
+                  if (err.code === 'ENOENT') return writeSymlink()
```

Comment:
Instead of doing a path exists check before attempting to delete, I just try to delete and if there's a `ENOENT`, we just proceed to write the symlink.